docker-compose.test.yml

```diff
@@ -194,6 +194,15 @@ services:
     command: celery worker -E -A inspirehep.celery --loglevel=INFO --purge --queues celery,migrator,harvests,orcid_push
     volumes_from:
       - test-static
+    healthcheck:
+      timeout: 5s
+      interval: 5s
+      retries: 5
+      test:
+        - "CMD"
```

Comment:
Why are we using sometimes `CMD` and sometimes `CMD-SHELL` ? I can't see the pattern.

Reply:
The difference is that CMD will run the command 'bare', with CMD-SHELL it will start the system shell, and run the given command there.
Usually if you want a specific shell (ex. bash) you should use CMD, if you don't care, but still want a shell, use CMD-SHELL, and if you don't need a shell, just CMD.

It's not a big issue to use CMD-SHELL when you could use  CMD, but the other way around might not work. Some features like `exec 3<> /dev/tcp/127.0.0.1/6379` or `[[]]` are bash specific though.